libs/entry-components/configure-theming.md

```diff
@@ -0,0 +1,435 @@
+# Theming Configuration Guide
+
+### Table of Contents
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Project Structure](#project-structure)
+- [Styles Configuration](#styles-configuration)
+- [Create custom theme](#create-custom-theme)
+- [Importing fonts](#importing-fonts)
+- [Theme Configuration Approaches](#theme-configuration-approaches)
+  - [Simplified Configuration](#1-simplified-configuration)
+    - [Configuration Properties](#2a-configuration-properties)
+      - [General Configuration](#general-configuration)
+      - [Tables Configuration](#tables-configuration)
+      - [Dialogs Configuration](#dialogs-configuration)
+  - [Native Angular Material Configuration](#2-native-angular-material-configuration)
+
+## Overview
+
+The **entry-components** library comes with a generator which simplifies project theme by eliminating the need for extensive custom styling. While the library comes with default settings, it provides the flexibility to be configured according to the unique requirements of each project. A wide range of properties can be leveraged to introduce varied style changes when configured appropriately. The guide provides a detailed walkthrough of this process.
+
+## Prerequisites
+
+Ensure that the following libraries are installed with the latest versions:
+
```

Comment:
Maybe we should specify preparing custom fonts (if consumers have such requirements) by putting them into assets and defining them with absolute path in font families (link with lower text on that topic).

We probably need one (or a few) link leading to angular material documentation. In order to digest theme, one needs to know at least something about angular material or design material at least.